src/migration.js

```diff
@@ -29,15 +30,16 @@ var validate = function(cb) {
         this.db.collection(this.collection).find({}, {}, {order : 1}).toArray(function(err, docs){
             assert.equal(err, null);
             var _steps = utilities.arrayToObject(this.steps, 'id');
-
-            docs.forEach(function(dbStep, index){
+            docs.forEach(function(dbStep){
+                var index = this.steps.findIndex(function(step){return step.id === dbStep.id});
                 if(this.steps[index]){
-                    this.steps[index].status = statuses.skipped;   
-
-                    if(!_steps[dbStep.id] || (dbStep.order && dbStep.order != _steps[dbStep.id].order)){
```

Comment:
when writing the test for ignoreOrder i realizes there's a problem with dbStep.order as it evaluates to false when order is `0`